### PR TITLE
Delete controller override and just override layout

### LINKED ISSUE #1815: Override CurationConcerns PermissionsController layout without copypasta
Sufia uses CurationConcerns' PermissionsController to handle confirming changes to GenericWork permissions and copying those down to FileSets. We override this currently only to set the layout to `nil` which gets the controller to pick up Sufia's layout. This approach introduces technical debt -- whenever CC makes changes to its PermissionsController, we'll need to copy/paste that over. This is not good. CurationConcerns already contains some machinery to deal with theming and layouts within controllers -- we should either use this in Sufia or find a way to make this work in Sufia. Another approach would be to put in a PR in CurationConcerns that extracts PermissionsController to PermissionsControllerBehavior. That way we should be able to pick up changes to the behavior and have a tiny overridden PermissionsController in Sufia.

### Related work

Refs #1731 (See comments for more discussion.)

### PR DESCRIPTION
**Pull request by cam156**
Fixes #1815 

Delete the controller override of CC that just sets the layout to nil.  This can be accomplished by overriding the 1_column layout with a copy of the sufia-one-column layout instead.

Also moves sufia-one-column to curation_concerns/1_column so we do not need to override the layout in all of the other controllers.

@projecthydra/sufia-code-reviewers

### CODE REVIEW

**Comment by jcoyne** [comment]
I'd prefer we keep an extra controller rather than having to manage an overridden view.

**Comment by cam156** [comment]
@jcoyne then you should comment on  #1815 to lobby to close it without change. I would think putting out own layout in would be something we should do.

**Comment by cam156** [comment]
After a conversation in slack it was determined to move the sufia-one-column layout to the curation_concern/1_column so that sufia can better tap into the curation concerns theming.  Working on an update to the PR

**Comment by cam156** [comment]
@jcoyne Better?

**Comment by awead** [comment]
Discussed this morning.

**Comment by awead** [comment]
@mjgiarlo Is there a way we can get the dependency check to run? We need this merged for the PSU sprint.

File: lib/generators/sufia/install_generator.rb [hunk 1]
```diff
@@ -87,12 +87,6 @@ def inject_sufia_application_controller_behavior
       end
     end
 
-    def use_blacklight_layout_theme
-      file_path = "app/controllers/application_controller.rb"
-      return unless File.exist?(file_path)
-      gsub_file file_path, /with_themed_layout '1_column'/, "layout 'sufia-one-column'"
```
Comment (jcoyne) [inline lib/generators/sufia/install_generator.rb]: can you add a note about this in https://github.com/projecthydra/sufia/issues/2491 ?

**Comment by jcoyne** [comment]
@cam156 Can we remove https://github.com/projecthydra/sufia/blob/master/app/views/layouts/sufia-one-column.html.erb now?

File: app/controllers/concerns/sufia/batch_uploads_controller_behavior.rb [hunk 2]
```diff
@@ -5,7 +5,6 @@ module BatchUploadsControllerBehavior
     include CurationConcerns::CurationConcernController
 
     included do
-      layout "sufia-one-column"
```
Comment (mjgiarlo) [inline app/controllers/concerns/sufia/batch_uploads_controller_behavior.rb]: Does this controller behavior not need `with_themed_layout '1_column'` or is that accounted for elsewhere?

File: app/controllers/concerns/sufia/works_controller_behavior.rb [hunk 3]
```diff
@@ -7,7 +7,6 @@ module WorksControllerBehavior
 
     included do
       self.show_presenter = Sufia::WorkShowPresenter
-      layout "sufia-one-column"
```
Comment (mjgiarlo) [inline app/controllers/concerns/sufia/works_controller_behavior.rb]: Does this controller behavior not need `with_themed_layout '1_column'` or is that accounted for elsewhere?

File: lib/generators/sufia/install_generator.rb (hunk 1 above)
Comment (mjgiarlo) [inline lib/generators/sufia/install_generator.rb]: :+1:

File: app/controllers/sufia/admin_sets_controller.rb [hunk 4]
```diff
@@ -1,7 +1,6 @@
 module Sufia
   class AdminSetsController < ApplicationController
     include CurationConcerns::CollectionsControllerBehavior
-    layout "sufia-one-column"
```
Comment (mjgiarlo) [inline app/controllers/sufia/admin_sets_controller.rb]: Is this picking up the themed layout from one of its dependencies?

File: app/controllers/concerns/sufia/works_controller_behavior.rb (hunk 3 above)
Comment (cam156) [inline app/controllers/concerns/sufia/works_controller_behavior.rb]: @mjgiarlo any controller that includes on inherits from CurationConcerns::CollectionsControllerBehavior gets `with_themed_layout '1_column'` by default.  No need to override or set again.

File: app/controllers/sufia/admin_sets_controller.rb (hunk 4 above)
Comment (cam156) [inline app/controllers/sufia/admin_sets_controller.rb]: @mjgiarlo any controller that includes on inherits from CurationConcerns::CollectionsControllerBehavior gets `with_themed_layout '1_column'` by default.  No need to override or set again.

File: app/controllers/concerns/sufia/batch_uploads_controller_behavior.rb (hunk 2 above)
Comment (cam156) [inline app/controllers/concerns/sufia/batch_uploads_controller_behavior.rb]: @mjgiarlo any controller that includes on inherits from CurationConcerns::CollectionsControllerBehavior gets `with_themed_layout '1_column'` by default.  No need to override or set again.

**Comment by cam156** [comment]
@jcoyne I moved it, so in essence it is getting removed.  That is all the sufia specific headers and menus so we want that code.  This PR just puts it in the place curation concerns will find it. from files changed: `app/views/layouts/sufia-one-column.html.erb → ...youts/curation_concerns/1_column.html.erb`

File: app/controllers/concerns/sufia/works_controller_behavior.rb (hunk 3 above)
Comment (mjgiarlo) [inline app/controllers/concerns/sufia/works_controller_behavior.rb]: Excellent, thanks!

**Pull request merged**